crates/rip_bin/src/main.rs

```diff
@@ -344,13 +91,23 @@ async fn main() {
 }
 
 /// Constructs a default [`EnvFilter`] that is used when the user did not specify a custom RUST_LOG.
-pub fn get_default_env_filter(verbose: bool) -> EnvFilter {
-    let mut result = EnvFilter::new("rip=info")
-        .add_directive(Directive::from_str("rattler_installs_packages=info").unwrap());
-
-    if verbose {
-        result = result.add_directive(Directive::from_str("resolvo=info").unwrap());
-    }
+pub fn get_default_env_filter(verbose: clap_verbosity_flag::Verbosity) -> EnvFilter {
+    // Always log info for rattler_installs_packages
```

Comment:
what do you mean by always log info for rattler_installs_packages?
I see that we are returning it always for rip

Reply:
Yeah its the same new but `rip` is kind of the alias, we wanted to use that but it was taken on crates.io, so the longer name is the official crate name.

Reply:
got it - thank you for explaining it